src/blocks/_pro/post-list/index.php

```diff
@@ -161,14 +161,16 @@ function vk_blocks_register_block_post_list() {
 	);
 
 	$the_taxonomies = get_taxonomies(
-		array(),
+		array(
+			'public' => true,
+		),
 		'objects',
 		'and'
 	);
 
 	$term_by_taxonomy_name = array();
 	foreach ( $the_taxonomies as $the_taxonomy ) {
-		$terms                                        = get_terms( $the_taxonomy->name, array( 'hide_empty' => false ) );
+		$terms                                        = array_values( get_terms( $the_taxonomy->name, array( 'hide_empty' => 1 ) ) );
```

Comment:
@drill-lancer 
```suggestion
		$terms                                        = array_values( get_terms( $the_taxonomy->name, array( 'hide_empty' => false ) ) );
```

ブロック配置時には単にまだ投稿を登録してないだけで、
対象のタクソノミーのチェックボックスは指定できた方が良いというケースもあるのでここはfalseに戻し。